### DISCUSSION TITLE
can not open in macos15.0

### DISCUSSION THREAD

**Discussion by aatrooox** [pawelsalawa/sqlitestudio#5169] [General]
### Details
i can not open /Volumes/SQLiteStudio/SQLiteStudio-3.4.4-osx-installer.app in macos15.0
### Steps to reproduce
click /Volumes/SQLiteStudio/SQLiteStudio-3.4.4-osx-installer.app
### Operating system
macos 15.0
### SQLiteStudio version
3.4.4

**Comment by pawelsalawa** [comment]
Right click, from menu pick "Open". The warning window will appear, but this time it will have option like "Open anyway" or something like that.
